### PR TITLE
Update license, etc.

### PR DESCRIPTION
**Pull request by joneit**
This PR to be merged by an Openfin contributor, please.

1. Update license to address complaints:
   * Issue #766: License seems to preclude using in a tool (as opposed to an app). See [LICENSE](768/files?utf8=✓&diff=split#diff-9879d6db96fd29134fc802214163b95a) change which attempts to clarify that the preclusion only intended for a tool that is nothing more than a "gird visualization engine."
   * Issue #743: Calling license "MIT" is misleading (see [package.json](768/files?utf8=✓&diff=split#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R17) change)
   * Changed coyright from "© Openfin 2015" to "© The Hypergrid Authors 2015-2018". (Alternative: "© Openfin 2015-2016, © 2017-2018 The Hypergrid Authors" ??)
2. Update Hyperblotter references in [README.md](768/files?utf8=✓&diff=split#diff-04c6e90faac2675aa89e2176d2eec7d8)
   * Replaced with link to "AdaptableBlotter" installer
   * Copied [image](https://openfinv2.wpengine.com/wp-content/uploads/2018/01/partner-adaptableblotter_image-01@2x-667x375@2x.png) from https://openfin.co/partners/
3. Removed all other references to openfin (besides README reference to AdaptableBlotter)
4. See also [this PR](https://github.com/fin-hypergrid/jsdoc-template-hypergrid/pull/1) which updates "© Openfin 2015" to "© The Hypergrid Authors 2015-2018" that appears at the bottom _of every page_ of the generated [API docs](https://fin-hypergrid.github.io/core/doc).

### CODE REVIEW

**Comment by Dwaynekj** [comment]
Will review tomorrow

File: package.json [hunk 1]
```diff
@@ -7,8 +7,14 @@
...
-  "author": "SWirts, JEiten, DJones, NMichaud",
-  "license": "MIT",
+  "author": "Steve Wirts (https://github.com/stevewirts)",
+  "contributors": [
+    "Jonathan Eiten (https://github.com/joneit)",
+    "Dwayne Jones (https://github.com/dwaynekj)",
+    "Naveen Michaud-Agrawal (https://github.com/nmichaud)",
+    "Hugo Nogueira (https://github.com/hugoeanogueira)"
+  ],
+  "license": "MIT (as modified in LICENSE file)",
```
Comment (mjosling) [inline package.json]: What does "as modified in License File" mean? MIT License is what it is, not clear how it can be modified?

**Comment by Dwaynekj** [comment]
Hey @joneit We want the copyright to reflect `© OpenFin and The Hypergrid Authors 2015-2018`

**Comment by Dwaynekj** [comment]
@joneit We are discussing internally with Chuck we will get back to you

**Comment by joneit** [comment]
@mjosling Per #766 It has been pointed out to us that calling it an MIT license is misleading because it is in fact modified. We could call it “Hypergrid” license.....

**Comment by Dwaynekj** [comment]
@joneit We want the copyright to be `© OpenFin and The Hypergrid Authors 2015-2018`

**Comment by joneit** [comment]
I made the change to copyright notice as requested (except I put the year first which is proper).

File: LICENSE [hunk 2]
```diff
@@ -13,8 +13,8 @@ The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
 
 You may not redistribute the software or modifications as part of any
-application that can be described as a development toolkit, library or
-framework.
+application that can be described as a grid visualization engine development
+toolkit, library or framework.
```
Comment (joneit) [inline LICENSE]: Comments on this? (Issue #766) If I understand the original intent correctly, people can use Hypergrid in their project, so long as it's not just a wrapper with a new name slapped on it passing it off as their own.

Adding the phrase "grid visualization engine" is my attempt to make it clear that their project can be a framework, just not solely and specifically a grid framework, with no value added.

Can you think of perhaps a better way of expressing this? Alternatives I considered were:
* "_specifically_ as a grid visualization engine [toolkit...]"
* "_solely and specifically_ as a grid visualization engine [toolkit...]"

Comment (Dwaynekj) [reply LICENSE]: Hey @joneit We want to use the original MIT License verbatim and not add custom language.

Comment (joneit) [reply LICENSE]: OK.... just to confirm, this is a *big* change, right? Just so we are on the same page, the custom paragraph in the LICENSE file **came from Openfin and has been in there forever:**

> You may not redistribute the software or modifications as part of any
application that can be described as a grid visualization engine development
toolkit, library or framework.

Removing the above paragraph entirely will allow anyone to fork it, rename it, and call it their own. I personally don't see that as a good idea, but it's your call!

Please confirm that you want to do this, which would revert it to a plain vanilla MIT license, and I will make the edit in a new commit.

Thx.

Comment (Dwaynekj) [reply LICENSE]: @joneit Just got confirmation. We want to go MIT all the way

**Pull request merged**